
    public ExecutionException(Throwable cause) {
        super(cause);
    }
}
